src/main/java/com/hackclub/hccore/commands/LocCommand.java

```diff
@@ -89,6 +88,29 @@ public boolean onCommand(CommandSender sender, Command cmd, String alias, String
                 }
                 break;
             }
+            // /loc rename <old name> <new name>
+            case "rename": {
+                if (args.length < 3) {
+                    sender.sendMessage(
+                            ChatColor.RED + "You must include both a location and a new name");
```

Comment:
It's not clear how to use this subcommand from the message. Change it to `/loc rename <old name> <new name>` (white)

Reply:
why you should return false

Reply:
@nice6599 Returning false only returns this: https://github.com/hackclub/HCCore/blob/3b3c4b393cec37fe0a6e4bffffd4815f357cd3b4/src/main/resources/plugin.yml#L16

It's not specific enough for this subcommand.